app/src/main/java/atorch/statspuzzles/PuzzleSelection.java

```diff
@@ -10,9 +10,11 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import thing.that.does.not.exist as ThisShouldProduceAnError
```

Comment:
This caused an error, as desired:

![Screenshot from 2022-07-09 09-08-45](https://user-images.githubusercontent.com/1395971/178113734-9eb291ed-275b-4210-a24b-85afbe777350.png)